### ISSUE TITLE
Allow edit of Operations in metaproject admin gui

### ISSUE THREAD

**Issue by NCIEVS** [NCIEVS/nci-protege5#534]
This is currently the only aspect that is not editable, it doesn't have the x and o icons to the right:

![image](https://user-images.githubusercontent.com/7783321/142651867-1273a920-b4fa-42b7-8ca7-4492a251fedc.png)

Any edits need to be done on the server config file.  This came up because the identifier in the metaproject / json config file  does not match the expected identifier for a new operation used in the metaproject plugin (metaproject/impl/Operations.java).  

The ability to edit identifiers will be a feature request in a separate ticket.

**Comment by bdionne** [comment]
As I look at the code, I think the mistake in this one is having the ability to add new operations in the GUI. We should remove that, and not allow the user to edit these.

The reason is that these operations are designed to be constants in the software. They are the atomic level operations based on the ontology objects. We allow edit and adding of roles as those are created by the user and are collections of these operations.

I think this issue arose because of the need to add the new operations to the config file. Mistakes were made and it was desired to fix them in the GUI. Given the way the code works I'm not sure adding this feature will work, and will likely create more problems.

In other words, these operations are the primitives and are linked to the software. Does that make sense?

**Comment by NCIEVS** [comment]
Yes, it makes sense.  But this is not to create/edit operations per se, but to match those operations with the appropriate constants in the config file, and edit that config file in the gui rather than in text and then have IT group deploy.  (An alternative that we discussed was the ability to edit the json from the admin gui.)

**Comment by singhik** [comment]
This ticket is intrinsically addressed via a workaround fix per  #556  . However, as the Admin GUI is not changed ,this ticket is kept open to analyze in Future, this was discussed in Team Meeting 9/15/2022 .
